eo-maven-plugin/src/main/resources/org/eolang/maven/transpile/to-java.xsl

```diff
@@ -171,6 +171,12 @@
       </xsl:otherwise>
     </xsl:choose>
   </xsl:function>
+  <!-- Escape `+` in test name syntax. -->
+  <xsl:function name="eo:escape-plus">
+    <xsl:param name="name"/>
+    <xsl:variable name="pos" select="string-length(tokenize($name, '\+')[1]) + 1"/>
+    <xsl:value-of select="concat(substring($name, 1, $pos - 1), substring($name, $pos + 1))"/>
```

Comment:
@h1alexbel maybe it's better just to:
```
if starts-with($name, '+') then substring($name, 1) else $name
```

Reply:
@maxonfjvipon I tried, but it won't work for location with class name like: `Φ.list-tests.+tests-iterates-with-eachi.φ.α0`